scripts/deploy_prod.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Ensure working directory is clean
+if ! git diff-index --quiet HEAD; then
+  echo "Working directory not clean, please commit your changes first"
+  exit
+fi
+
+# Merge master into master_clever, then push master_clever
+# Deployment to Clever Cloud is actually triggered via a hook
+# on a push on this branch
+git checkout master
+git pull origin master
+git checkout master_clever 
+git pull origin master_clever 
+git merge master --no-edit
```

Comment:
Dans ma version perso j'avais un `--no-ff` en plus mais je n'arrive plus à retrouver pourquoi je l'ai mis. On peut oublier du coup.

https://stackoverflow.com/questions/9069061/what-is-the-difference-between-git-merge-and-git-merge-no-ff

Reply:
Ça vaut le coup de normaliser le `--no-ff` qui permet de faire un rollback d’un déploiement entier plus facilement que sans
![image](https://user-images.githubusercontent.com/1223316/123266657-8699b700-d4fc-11eb-8eed-4de91172ffe4.png)

Reply:
Ok nice! je te laisse décider.